packages/database/src/core/PersistentConnection.ts

```diff
@@ -184,6 +193,29 @@ export class PersistentConnection extends ServerActions {
     }
   }
 
+  get(query: Query, onComplete: (a: string, b: unknown) => void) {
```

Comment:
This should:
- Specify a return type
- Ideally return a Promise
- If not, use descriptive names for the options argument (and `b` seems to always be a string?)